salt/cloud/clouds/azurearm.py

```diff
@@ -1173,10 +1180,6 @@ def request_instance(vm_):
                 sku=img_sku,
                 version=img_ver,
             )
-        if win_installer:
-            os_type = 'Windows'
-        else:
-            os_type = 'Linux'
```

Comment:
I am ok with the other change, but if os_type is not specified, I would like to leave this as the default, and log a debug message instead of raising a system exit error.